lib/scanner/walk_test.go

```diff
@@ -366,6 +366,43 @@ func TestWalkSymlinkWindows(t *testing.T) {
 	}
 }
 
+func TestWalkRootSymlink(t *testing.T) {
+
+	// Create a folder with a symlink to it
```

Comment:
Too many blanks lines everywhere, esp around comments from both sides.

Reply:
This function was mostly a copy/paste of the TestWalkSymlink[Windows/Unix]() functions with modifications but mostly retained style/spacing. I left the pull request open to edits from maintainers so it would probably be easiest for you to shore up styling.